Lib/fontbakery/specifications/general.py

```diff
@@ -150,45 +152,50 @@ def com_google_fonts_check_037(font):
       if "Table Test:" in line:
         continue
       filtered_msgs += line + "\n"
-    yield INFO, ("Microsoft Font Validator returned an error code."
-                 " Output follows :\n\n{}\n").format(filtered_msgs)
+    yield INFO, ("Microsoft Font Validator returned an error code. "
+                 "Output follows :\n\n{}\n").format(filtered_msgs)
```

Comment:
I don't like this style. Please revert, since it does not match the style used in all the rest of the code.